### ISSUE TITLE
Bitch Sisters ga Seijun na Hazu ga Nai!!? Viewing issue

### ISSUE THREAD

**Issue by AlchemistofAtlas** [morkt/GARbro#253]
Good evening; it's been a while since last time I asked for something so here it is. I once asked to give support to this game; while it worked fine the time you gave it support, now it no longer works with recent versions of GARBro (nor with version 1.5.35 which was the latest one).

It gives me this error: 
![garbro issue](https://user-images.githubusercontent.com/20749122/48659378-93d19980-ea15-11e8-9659-4632e77e9358.png)

Is there a way to solve this? I purchased the game with a special patch included that I want to see through it. Thanks beforehand.

VNDB: https://vndb.org/v20429

**Comment by AlchemistofAtlas** [comment]
In addition, I have the minimum required framework version for GARBro, so, I don't know what's the issue that prevents GARBro from further opening these files. It says invalid data to decode.

**Comment by morkt** [comment]
let me get this straight:
1. garbro was working fine.
2. you've installed some patch on top of the game.
2. garbro doesn't display images anymore.

is that correct?

**Comment by AlchemistofAtlas** [comment]
Correct. Installed a patch that includes different spriteworks for the charcters, something that is of my interest to extract.

GARbro works with other file formats from other games just fine, but not with games that use ShiinaRio format (version 2.50 if I'm correct); at least, not with this game. This problem even came before even I could purchase the game with the patch I wanted to add in previous versions.

**Comment by morkt** [comment]
if you're still interested provide more information about this patch: is it official? what's its version? how did you get it? this info should have been provided in the very first post tbh.

from what i can tell, all official patches available at developer's site are supported.

**Comment by AlchemistofAtlas** [comment]
Yes. Indeed I'm still interested.

The special patch is official (includes lingerie sprites), but said patch is only included to those who purchase the game on DMM.com (Download version, that is).

DL version comes under its latest version from my (admittedly limited) knowledge, which is 1.01 (I think so). It's not necessary to download the correction patch from onomatope's site because it only applies to the package version.

That's all I know from this point. A partner of mine who got the patch as well said that problem with GRBbro happened to him as well. The main theory is that they may be use a different encryption method for a patch.

**Comment by AlchemistofAtlas** [comment]
I don't know if you may find this, but, I finally found the problem. The extraction works, but ONLY in the Package version of the game, not on the Download version.
